WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -268,11 +268,22 @@ private static SimulationStatistics GetSimulationStatistics(Simulation simulatio
         {
             var statistics = SimulationStatistics.FromServiceModel(simulation.Statistics);
 
-            if (statistics != null)
+            if (statistics != null && simulation.ActualStartTime.HasValue)
```

Comment:
Please add a comment explaining why you're checking for simulations that have a value for ActualStartTime. What happens if I pass in a simulation that isn't running?

Reply:
sure added